sdk/ai/azure-ai-projects/azure/ai/projects/telemetry/agents/_ai_agents_instrumentor.py

```diff
@@ -392,67 +578,7 @@ def _status_to_string(self, status: Any) -> str:
         return status.value if hasattr(status, "value") else status
 
     def _add_tool_assistant_message_event(self, span, step: RunStep) -> None:
-        tool_calls = [
-            {
-                "id": t.id,
-                "type": t.type,
-                "function": (
-                    {"name": t.function.name, "arguments": json.loads(t.function.arguments)}
-                    if isinstance(t, RunStepFunctionToolCall)
-                    else None
-                ),
-            }
-            for t in cast(RunStepToolCallDetails, step.step_details).tool_calls
-        ]
-
-        attributes = self._create_event_attributes(
-            thread_id=step.thread_id,
-            agent_id=step.agent_id,
-            thread_run_id=step.run_id,
-            message_status=step.status,
-            usage=step.usage,
-        )
-
-        if _trace_agents_content:
-            attributes[GEN_AI_EVENT_CONTENT] = json.dumps({"tool_calls": tool_calls}, ensure_ascii=False)
-        else:
-            tool_calls_non_recording = self._remove_function_call_names_and_arguments(tool_calls=tool_calls)
-            attributes[GEN_AI_EVENT_CONTENT] = json.dumps({"tool_calls": tool_calls_non_recording}, ensure_ascii=False)
-        span.span_instance.add_event(name="gen_ai.assistant.message", attributes=attributes)
```

Comment:
@lmolkova In Semantic Conventions, do we have both names gen_ai.assistant and gen_ai.agent in attributes/events?

Reply:
Possible change here can in separate PR

Reply:
yes, we have both - the `gen_ai.assistant.message` is also used for inference, where the role is `assistant`.

the agent is used for agents/assistant API. I see why there might be some confusion, but that's not something we can easily change.